### ISSUE TITLE
[iOS + Android] Range or years for vehicle information

### ISSUE THREAD

**Issue by nguyendtt** [trietphamvm/DAR#95]
I'm noticing that we have a wide range of years for the vehicle information screen, from something like 1900 - 2030+.  I think the newest year should only be the current year + 1 (ex. the current max year would be 2016).  The oldest year should be 2000 (16 year old?).  @trietphamvm @phuong37

**Comment by phuong37** [comment]
i agree with chi @nguyendtt - if we do any older, we risk the cars having more mechanical issues. thanks!

**Comment by phuong37** [comment]
chi @nguyendtt - hold off on this while i get confirmation from the team. thanks!
